docs/user/overview.md

```diff
@@ -0,0 +1,61 @@
+# Installer Overview #
+
+The OpenShift Installer is designed to help users, ranging from novices to experts, create OpenShift clusters in various environments. By default, the installer acts as an installation wizard, prompting the user for values that it cannot determine on its own and providing reasonable defaults for everything else. For more advanced users, the installer provides facilities for varying levels of customization.
+
+In [supported environments](#supported-environments), the installer is also capable of provisioning the underlying infrastructure for the cluster. It is recommended that most users make use of this functionality in order to avoid having to provision their own infrastructure themselves. In unsupported environments or scenarios in which the created infrastructure is incompatible, the installer can stop short of creating the infrastructure, allowing the user to provision their own infrastructure using the cluster assets generated by the installer.
```

Comment:
nit: You can drop "themselves" without changing the meaning.

Also "the created infrastructure is incompatible" -> "installer-created infrastructure would be incompatible"?

", allowing the user" -> "and allow the user"